rpcs3/Emu/Cell/Modules/cellSsl.cpp

```diff
@@ -23,14 +23,22 @@ s32 cellSslEnd()
 	return CELL_OK;
 }
 
-std::string getCert(const std::string certPath, const int certID)
+std::string getCert(const std::string certPath, const int certID, const bool isNormalCert = false)
 {
-	//Stupid cellssl
 	int newID = certID;
-	if (certID == 6)
-		newID = 23;
-	if (certID > 6 && certID < 23)
-		newID = certID - 1;
+
+	// The 'normal' certs have some special rules for loading.
+	if (isNormalCert)
+	{
+		if (certID == 6)
+			newID = 23;
+		if (certID > 6 && certID <= 23)
+			newID = certID - 1;
+		if (certID == 13)
+			newID = 6;
+		if (certID == 18)
+			newID = 15;
+	}
```

Comment:
Why not if else? Why not switch case?

Reply:
Not `switch case` because you have to do the `break;` for each case which adds more lines than it's worth in this case.

I'll change it to `if else` though.